qgispluginci/release.py

```diff
@@ -563,7 +565,7 @@ def release(
         parameters,
         release_version,
         archive_name,
-        add_translations=tx_api_token is not None,
+        add_translations=add_translations,
```

Comment:
Strange this new variable. Maybe I'm tired, but I fail to understand how `tx_api_token` cannot be used, when I see how the `add_translations` is set.